src/Knp/Snappy/AbstractGenerator.php

```diff
@@ -630,8 +630,16 @@ protected function prepareOutput($filename, $overwrite)
         }
 
         $scheme = isset($parsedFilename['scheme']) ? \mb_strtolower($parsedFilename['scheme']) : '';
-        if ($scheme !== '' && $scheme !== 'file') {
-            throw new InvalidArgumentException(\sprintf('The output file scheme is not supported. Expected \'\' or \'file\' but got \'%s\'.', $scheme));
+        if (!(
+            $scheme === '' ||
+            $scheme === 'file' ||
+            (
+                // Check if it's a Windows path
+                \strlen($scheme) === 1 &&
+                \preg_match('/^[a-z]:(?:[\\\\\/]?(?:[\w\s!#()-]+|[\.]{1,2})+)*[\\\\\/]?/i', $filename) === 1
```

Comment:
Can you give some exemples with specs for example ? Maybe set the regex in a constant will clarify the condition.